add-on/panels/css/slideshow.css

```diff
@@ -0,0 +1,179 @@
+@import url('http://fonts.googleapis.com/css?family=Open+Sans');
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+html {
+  font-size: 10px;
+  font-family: menu;
+  color: #fff;
+}
+
+body {
+  background: none;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  line-height: 2.4rem;  /* or 1.3rem original*/
+}
+.slideshow {
+
+}
+.slides {
+  display: block;
+  width: 100%;
+  height: 390px;
+  overflow: hidden;
+  white-space: nowrap;
+}
+.slide {
+  display: none;
+  height: 100%;
+  width:  100%;
+}
+.slide.slide--active {
+  display: block;
+}
+
+.control-panel {
+  height: 60px;
+  background: #fff;
+  width: 100%;
+}
+
+.controls {
+}
+
+.toggle {
+  color: white;
+  display: block;
+  padding: 0px;
+  position: absolute;
+  bottom: 20px;
+  background-color: transparent;
+  background-image: url(../../shared/img/arrow-01.svg);
+  background-repeat: no-repeat;
+  background-size: 20px 20px;
+  border: none;
+  /*padding: 0;*/
+  height: 20px;
+  width: 20px;
+}
+.toggle-prev {
+  left: 20px;
+  transform: scaleX(-1);
+}
+.toggle-next {
+  right: 20px;
+
+}
+
+.button-close {
+  display: block;
+  position: absolute;
+  background-color: transparent;
+  background-image: url(../../shared/img/close-02.svg);
+  background-repeat: no-repeat;
+  background-size: 20px 20px;
+  border: none;
+  padding: 0;
+  height: 20px;
+  width: 20px;
+  top: 20px;
+  right: 20px;
+}
+
+
+/* slide content */
+.slide-layout {
+  padding-top: 50px;
+  padding-left: 50px;
+  padding-bottom: 20px;
+  width: 600px;
+}
+
+.slide-layout h2 {
+  font-family: "Open Sans";
+  /*font-size: 25pt;*/
+  font-size: 2.8rem;
+  font-weight: 600;
+  white-space: normal;
+  color: #fff;
+  line-height: 3rem;
+  margin-bottom: 10px;
+}
+
+.slide-layout .slide-text {
+  font-family: "Open Sans";
+  /*font-size: 18pt;*/
+  font-size: 2rem;
+  white-space: normal;
+  color: #fff;
+  line-height: 2.8rem;
+}
+
+.slide img {
+  display: block;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  border: none;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 0;
+  float: right;
+  margin-top: -20px;
+}
+.slide1 {
+  height: 100%;
+  width:  100%;
```

Comment:
`.slide` already has height & width of 100%, so I don't think these are necessary.